ext/hal/nordic/nrfx_config_nrf9160.h

```diff
@@ -193,6 +193,11 @@
         ((NRF_UARTE_Type *)DT_NORDIC_NRF_UARTE_UART_3_BASE_ADDRESS)
 #endif
 
+#ifdef DT_INST_0_NORDIC_NRF_UICR_BASE_ADDRESS
```

Comment:
This won't work in 1.14. We need to use `DT_NORDIC_NRF_UICR_UICR_BASE_ADDRESS` and add `uicr` alias to _dts/arm/nordic/nrf9160.dtsi_.